### PR TITLE
Make ACL Docs Index available on public demo

### PR DESCRIPTION
**Pull request by sbhaktha**
@dirkgr : Change per Oren. He wants us to show the ACL papers index in the public demo, and have only that be selected by default. The prod deploy configuration is changing appropriately in a separate PR in the `deploy` repo -- coming up.

### CODE REVIEW

File: src/main/resources/all-corpora.conf [hunk 1]
```diff
@@ -7,7 +7,7 @@ IkeToolWebapp = {
       description = "All the ACL papers (about 20k). Indexed in memory or Saruman."
       item = {
         type = directory
-        datastore = private
+        datastore = public
```
Comment (dirkgr) [inline src/main/resources/all-corpora.conf]: I'm assuming you uploaded this there as well?

Comment (sbhaktha) [inline src/main/resources/all-corpora.conf]: yup

**Comment by sbhaktha** [comment]
actually we keep the `SearchApps` wrapped in a `Future`, so we would have to map it every time to get the default setting. I think this is better.

**Comment by dirkgr** [comment]
No, it's not better. Look at how much code you added to support that one level of indirection. And you only need that setting in two places! In one of them, it's already taken care of because it filters by the search apps that are ready. In the second place, you can just do the same thing we're doing in the first place.

**Comment by sbhaktha** [comment]
I see what you're saying. Done! I'll merge unless you have something else.

File: src/main/resources/application.conf [hunk 2]
```diff
@@ -43,9 +61,9 @@ Tablestore = {
     // link, `user` and `password`.
     // AI2 Internal users: use the database in the `test` deploy environment and the same username. You can get the password
     // from this file in the ops-keystore in S3.
-    url = "jdbc:postgresql://"${POSTGRES_DB}":5432/okcorpus"
+    url = "jdbc:postgresql://okcorpus-dev.civm9kvdku8h.us-west-2.rds.amazonaws.com:5432/okcorpus"
     user = "okcorpus"
-    password = ${POSTGRES_PASSWORD}
+    password = "InteractiveKEtest"
```
Comment (dirkgr) [inline src/main/resources/application.conf]: I don't think we can have passwords and hostnames in here. In fact, you might have to change the password now. Isn't IKE open source? You should change it immediately!

File: src/main/scala/org/allenai/ike/IkeToolWebapp.scala [hunk 3]
```diff
@@ -98,18 +94,14 @@ class IkeToolActor extends Actor with HttpService with SprayJsonSupport with Log
 
   val serviceRoute = pathPrefix("api") {
     parameters('corpora.?) { corpora =>
+      val readySearchAppsWithDefaultSelected =
+        searchApps.filter(app => app._2.isCompleted && app._2.get.defaultSelected)
```
Comment (dirkgr) [inline src/main/scala/org/allenai/ike/IkeToolWebapp.scala]: If you want, you could use pattern matching and write `searchApps.filter { case (_, app) => app.isCompleted && app.defaultSelected }`, but this is fine, too.

**Pull request merged**